core/modules/path/path.path.inc

```diff
@@ -14,8 +14,8 @@
 function node_path_info() {
   $info['node'] = array(
     'entity type' => 'node',
-    'label' => t('Content paths'),
-    'pattern description' => t('Default path pattern (applies to all content types with blank patterns below)'),
+    'label' => t('Content'),
```

Comment:
The problem is that the noun changes depending on which admin page we are on. For example, in some cases the noun is "URLs" and in some places the noun is "URL patterns". See
```php
'#title' => t('!label URL aliases', array('!label' => $path_info['label'])),
```
```php
$group_header = t('!label URL patterns', array('!label' => $path_info['label']));
```